test/passing/doc_comments.mli.ref

```diff
@@ -403,3 +403,6 @@ end
 
 (** {e foooooooo oooooooooo ooooooooo ooooooooo} {{!some ref} fooooooooooooo
     oooooooo oooooooooo} {b fooooooooooooo oooooooooooo oooooo ooooooo} *)
+
+(** foooooooooooooooooooooooooooooooooooooooooooooooooo foooooooooooo {b eee
+    \+ eee eee} *)
```

Comment:
Could you also add a test with the word to escape (`+`) is the first in the `{}` ? As it involves the `space_elt` hack.

Reply:
done